github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -353,7 +353,7 @@ presubmits:
         name: ""
         resources:
           requests:
-            memory: 4Gi
+            memory: 8Gi
```

Comment:
This is not run by default, so doubling it's memory request will have no effect.

Reply:
I think it is ok at 8Gi - just so if the test lane is scheduled - it is not squeezed onto a node that already has a number of jobs running.